src/aics-image-viewer/components/ChannelsWidgetRow/index.tsx

```diff
@@ -158,74 +142,58 @@ export default class ChannelsWidgetRow extends React.Component<ChannelsWidgetRow
-    const { index, saveIsosurface } = this.props;
-    saveIsosurface(index, format);
   };
 
-  renderSurfaceControls = (): React.ReactNode => (
+  const renderSurfaceControls = (): React.ReactNode => (
     <Col span={24}>
       <h4 className="ant-list-item-meta-title" style={{ marginTop: "20px", marginBottom: "5px" }}>
         Surface settings:
       </h4>
-      {this.createSliderRow("Isovalue", 255, ISOVALUE_DEFAULT, this.onIsovalueChange)}
-      {this.createSliderRow(
+      {createSliderRow("Isovalue", 255, ISOVALUE_DEFAULT, onIsovalueChange)}
+      {createSliderRow(
         "Opacity",
         ISOSURFACE_OPACITY_SLIDER_MAX,
         ISOSURFACE_OPACITY_DEFAULT * ISOSURFACE_OPACITY_SLIDER_MAX,
-        this.onOpacityChange
+        onOpacityChange
       )}
-      <Button
-        disabled={!this.props.isosurfaceChecked}
-        onClick={this.createSaveIsosurfaceHandler("GLTF")}
-        style={STYLES.raisedButton}
-      >
+      <Button disabled={!isosurfaceChecked} onClick={() => saveIsosurface(index, "GLTF")} style={STYLES.raisedButton}>
         Save GLTF
       </Button>
-      <Button
-        disabled={!this.props.isosurfaceChecked}
-        onClick={this.createSaveIsosurfaceHandler("STL")}
-        style={STYLES.raisedButton}
-      >
+      <Button disabled={!isosurfaceChecked} onClick={() => saveIsosurface(index, "STL")} style={STYLES.raisedButton}>
         Save STL
       </Button>
     </Col>
   );
 
-  renderControls = (): React.ReactNode => (
+  const renderControls = (): React.ReactNode => (
     <div style={STYLES.settingsContainer}>
-      {this.props.volumeChecked && (
+      {volumeChecked && (
         <Row justify="space-between" className="volume-settings">
           <h4 className="ant-list-item-meta-title">Volume settings:</h4>
-          {this.createTFEditor()}
+          {createTFEditor()}
         </Row>
       )}
-      {this.props.isosurfaceChecked && <Row justify="space-between">{this.renderSurfaceControls()}</Row>}
+      {isosurfaceChecked && <Row justify="space-between">{renderSurfaceControls()}</Row>}
     </div>
   );
 
-  render(): React.ReactNode {
-    const rowClass = this.state.controlsOpen ? "row-card" : "row-card controls-closed";
-    return (
-      <List.Item key={this.props.index} className={rowClass} actions={this.renderActions()}>
-        <List.Item.Meta
-          title={<span style={STYLES.channelName}>{this.props.name}</span>}
-          avatar={this.createColorPicker()}
-        />
-        {this.state.controlsOpen && this.renderControls()}
-      </List.Item>
-    );
-  }
-}
+  const rowClass = controlsOpen ? "row-card" : "row-card controls-closed";
+  return (
+    <List.Item key={index} className={rowClass} extra={renderActions()}>
+      <List.Item.Meta title={<span style={STYLES.channelName}>{props.name}</span>} avatar={createColorPicker()} />
+      {controlsOpen && renderControls()}
+    </List.Item>
+  );
+};
```

Comment:
I decided to limit this PR to making the control panel look like the design as long as the user didn't open the advanced settings, since there's enough to change in there for another PR or few. I can slip this change in since I agree it's one of the biggest missing pieces, but this can also be its own small PR later.

Reply:
I don't mind leaving it for later if we document it in an issue ticket or something.

Reply:
It's captured broadly by #80, though that could have sub-tickets.